modules/webhook/lambdas/webhook/src/webhook/handler.ts

```diff
@@ -24,33 +25,53 @@ export async function handle(headers: IncomingHttpHeaders, body: string): Promis
     return response;
   }
   const payload = JSON.parse(body);
-  logger.info(`Received Github event ${githubEvent} from ${payload.repository.full_name}`);
+  LogFields.fields.event = githubEvent;
+  LogFields.fields.repository = payload.repository.full_name;
+  LogFields.fields.action = payload.action;
+
+  if (!supportedEvents.includes(githubEvent)) {
+    logger.warn(LogFields.fields, `Unsupported event type.`);
+    return {
+      statusCode: 202,
+      body: `Ignoring unsupported event ${githubEvent}`,
+    };
+  }
+
+  LogFields.fields.name = payload[githubEvent].name;
+  LogFields.fields.status = payload[githubEvent].status;
+  LogFields.fields.started_at = payload[githubEvent]?.started_at;
+
+  /*
+  The app subscribes to all `check_run` and `workflow_job` events.
+  If the event status is `completed`, log the data for workflow metrics.
+  */
+  LogFields.fields.completed_at = payload[githubEvent]?.completed_at;
+  LogFields.fields.conclusion = payload[githubEvent]?.conclusion;
 
   if (isRepoNotAllowed(payload.repository.full_name)) {
-    console.warn(`Received event from unauthorized repository ${payload.repository.full_name}`);
+    logger.error(LogFields.fields, `Received event from unauthorized repository ${payload.repository.full_name}`);
```

Comment:
I think by a small change we can keep the log much cleaner and also readable. 

So would suggest to add an function to `logger.ts`
```TypeScript
  public static print(): string {
    return ' LogFields: ' + JSON.stringify(LogFields.fields);
  }
 ```
And next add the logfiels at the end of the line, for example 

```suggestion
    logger.error(`Received event from unauthorized repository ${payload.repository.full_name}. `, LogFields.fields);
```